activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -1051,9 +1051,9 @@ def columns_for_distinct(columns, orders) # :nodoc:
       end
 
       # Adds timestamps (+created_at+ and +updated_at+) columns to +table_name+.
-      # Additional options (like <tt>null: false</tt>) are forwarded to #add_column.
+      # Additional options (like <tt>null: true</tt>) are forwarded to #add_column.
```

Comment:
Actually I think both are true ; this method forward all the given options, it just sets `:null` to `false` if the option isn't yet specified. What do you think about writing `Additional options (like +:null+) are forwarded` ?